src/cc1101.h

```diff
@@ -232,10 +232,10 @@ enum RFSTATE
 #define CC1101_DEFVAL_FREQ2_868  0x21        // Frequency Control Word, High Byte
 #define CC1101_DEFVAL_FREQ1_868  0x62        // Frequency Control Word, Middle Byte
 #define CC1101_DEFVAL_FREQ0_868  0x76        // Frequency Control Word, Low Byte
-// Carrier frequency = 902 MHz
-#define CC1101_DEFVAL_FREQ2_915  0x22        // Frequency Control Word, High Byte
-#define CC1101_DEFVAL_FREQ1_915  0xB1        // Frequency Control Word, Middle Byte
-#define CC1101_DEFVAL_FREQ0_915  0x3B        // Frequency Control Word, Low Byte
+// Carrier frequency = 900 MHz
```

Comment:
Lower frequency of 915 MHz ISM band is 902 MHz (902 MHz to 928 MHz). Do not send out of band.